src/view.rs

```diff
@@ -818,15 +839,15 @@ pub fn view_input_grammar(
                         let rect = target.get_bounding_client_rect();
                         (rect.get_width() - e.offset_x(), rect.get_height() - e.offset_y())
                     };
-                    info!{"offset: {} {}", offset_x, offset_y};
+                    // info!{"offset: {} {}", offset_x, offset_y};
                     let draggable_area = 4.0;
                     if offset_x < draggable_area  || offset_y < draggable_area {
                         Action::Resize(ResizeMsg::Start(drag_coord.clone()))
                     } else {
                         Action::Noop
                     }
                 })>
-                { value }
+                // { value }
```

Comment:
```suggestion
                // WORKAROUND: we don't bind `{ value }` here because the Vdom update triggered by ChangeInput does not overwrite it and leaves duplicate text in it's place. See this bug in the trello: https://trello.com/c/50CSe1Mm
```